vignettes/fallback.Rmd

```diff
@@ -70,18 +67,18 @@ duckdb %>%
   explain()
 ```
 
-The plan shows three operations:
+The plan shows three **operations**: 
```

Comment:
At most one per article, please, and really only if necessary 📣

Reply:
Hehe I'll keep the bold for the R-hub blog for instance then. I find it easier for skimming (reading once, then skimming to remind myself of important points) compared to italic. :innocent: